cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,22 @@
+from time import sleep
+
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.wait_for_db()
+
+    def wait_for_db(self):
```

Comment:
It's a good practice to separate the logic into a different method like `wait_for_db`. This improves the readability and maintainability of your code.